spec/helpers/pages_helper_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
```

Comment:
I'm not sure about making "Helper" tests.
Please remove this file.

BTW: You can blocking generating 'helper' files and also 'javascript' and 'css' files.
'6 Customizing Your Workflow by Changing Generators Templates'
http://guides.rubyonrails.org/generators.html